vendor/assets/stylesheets/ustyle/components/_comp-table-row.scss

```diff
@@ -0,0 +1,301 @@
+//             <li class="us-ct-row__criteria-li"><svg role="presentation" class="us-icon us-icon--small us-icon--tick us-ct-row__criteria-tick"><use xlink:href="/images/icons.svg#icon-tick"></use></svg>not be registered bankrupt in past 18 months or have proceedings</li>
+//             <li class="us-ct-row__criteria-li"><svg role="presentation" class="us-icon us-icon--small us-icon--tick us-ct-row__criteria-tick"><use xlink:href="/images/icons.svg#icon-tick"></use></svg>have no CCJs (past 12 months)</li>
+//           </ul>
+//           <p class="us-ct-row__criteria-additional">Additional criteria for acceptance from the lender may apply.</p>
+//         </div>
+//       </div>
+//     </div>
+//   </div>
+
+.us-ct-row {
+  position: relative;
+  display: block;
+  margin-bottom: 15px;
+  color: $c-navy;
+  background: #fff;
+  border: 1px solid $c-keylinegrey;
+  border-bottom-width: 5px;
+  border-radius: 5px;
+
+  &:hover {
+    cursor: pointer;
+  }
+
+  &__title {
+    display: none;
+    width: 100%;
+    padding: 15px;
+    border-bottom: 1px solid $c-keylinegrey;
+  }
+
+  &__title-text {
+    margin: 0 15px 0 0;
+    font-weight: bold;
+  }
+
+  &__product {
+    position: relative;
+    border-bottom: 1px solid $c-keylinegrey;
+  }
+
+  &__image {
+    max-width: 150px;
+  }
+
+  &__title--mobile {
+    margin-top: 10px;
+  }
+
+  &__usp--mobile {
+    display: flex;
+    margin-top: 10px;
+  }
+
+  &__col {
+    padding: 15px;
+    border-top: 1px solid $c-keylinegrey;
+    border-right: 1px solid $c-keylinegrey;
+
+    &--stretch {
+      display: flex;
+      align-self: stretch;
+      justify-content: space-between;
+    }
+
+    &--apply {
+      position: absolute;
+      top: 0;
+      right: 0;
+      width: calc(100% - 165px);
+      max-width: 200px;
+      text-align: center;
+    }
+  }
+
+  &__more-info {
+    display: inline-block;
+    margin-top: 10px;
+    color: #008fe9;
+    text-decoration: underline;
+    white-space: nowrap;
+    cursor: pointer;
+  }
+
+  &__rep-ex {
+    display: flex;
+    text-align: center;
+    border-bottom: 1px solid $c-keylinegrey;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__key-details-toggle {
+    display: block;
+    width: 100%;
+    padding: 10px;
+    font-size: 16px;
+    font-weight: 700;
+    color: #0c72bf;
+    text-align: center;
+    cursor: pointer;
+    background: none;
+    border: 0;
+  }
+
+  &__key-details-chevron {
+    width: 20px;
+    height: 20px;
+    padding: 3px;
+    margin-left: 4px;
+    border: 1px solid;
+    border-radius: 5px;
+    transform: translateY(5px) rotate(90deg);
+    stroke-width: 10px;
+    transform-origin: center;
+    fill: #0c72bf;
+
+    .us-ct-row__key-details-toggle-open & {
+      transform: translateY(5px) rotate(-90deg);
+    }
+  }
+
+  &__key-details-info {
+    padding: 0 15px;
```

Comment:
`15px` is `$gutter-width / 2` which is our default grid gutter

Reply:
Updated!